.github/workflows/execution-plan-snippet.yml

```diff
@@ -70,6 +70,10 @@ jobs:
         packages: write
         contents: read
     steps:
+      - name: Install Earthly
```

Comment:
There is a github action that installs earthly. You can find it [here](https://github.com/freiheit-com/kuberpult/blob/main/.github/workflows/commitlint.yml#L12). You should use that instead.